app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+import datetime
+
+
+class Cafe:
+    def __init__(self, cafe_name: str) -> None:
+        self.cafe_name = cafe_name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated")
+
+        vaccine_info = visitor["vaccine"]
+        expiration_date = vaccine_info["expiration_date"]
+        if expiration_date < datetime.date.today():
```

Comment:
There is no need to define multiple arguments, perform retrieval in the if condition instead